### DISCUSSION TITLE
Support for https://plus.rtl.de/

### DISCUSSION THREAD

**Discussion by rfq-mtorg** [aclap-dev/video-downloadhelper#996] [Help and Q&A]
### Your question

When downloading from https://plus.rtl.de/ , the downloaded video "blank" with no sounds.

Version: 9.0.2.12
Chrome: Version 126.0.6478.127 (Official Build) (arm64)
MacOS: 14.5 (23F79)

Sample:
https://plus.rtl.de/video-tv/shows/punkt-6-928640/2024-7-1006189/episode-132-sendung-vom-10072024-966466

![Screenshot 2024-07-11 at 1 09 23 PM](https://github.com/aclap-dev/video-downloadhelper/assets/147429330/b72a984a-47bd-42cd-8686-42e7b99e497e)

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"<removed>"}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "/Users/<removed>/Downloads",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 5,
    "successfull_dl": 98
}

**Comment by jcfrog** [comment]
Hi,
Sorry it won't work, they use DRM
<img width="259" alt="image" src="https://github.com/aclap-dev/video-downloadhelper/assets/368367/70b51b07-f2ca-48df-bb2b-4753f089c822">

**Reply by rfq-mtorg** [reply]
It can play via browser but cannot capture? Is that how DRM work? Is there no "screen-recorder" style option?

**Comment by rfq-mtorg** [comment]
It can play via browser but cannot capture? Is that how DRM work? Is there no "screen-recorder" style option?

**Reply by jcfrog** [reply]
No, it's encrypted and only the browser can play it.
It would also be illegal.

**Reply by rfq-mtorg** [reply]
Thank you for checking.

**Reply by GrampaWildWilly** [reply]
Screen recorder is a good idea but let me soften you up to the reality of things.  A few months ago I went through the exercise of installing about a dozen different screen recorders.  Not a single one of them was able to record my screen when I was playing content protected by DRM.  As soon as I opened any of the recorder applications, the screen where playback had been running fine a moment before suddenly turned into a black blob.  It is my understanding that DRM is now implemented in hardware, in a combination of your CPU, your graphics adapter, & the chipset on your motherboard.  There has been massive collusion to make sure content protected by DRM cannot be downloaded & cannot even be recorded.  It can be played in a browser, but you can't get a copy of it to play at will any time you like.  On my old computer where I used to run Windows 7 & the OBS screen recorder, I used to be able to record anything, including content protected by DRM.  But that was an old computer that did not have all this anti-competitive hardware, which apparently is "new."  New is relative.  I put together that old computer in late 2015.  Some time since then, the new hardware has become the standard.  You may have guessed by now that I have recently (about 8 months ago) replaced that old computer & now I'm on W11.  Progress means losing old functionality.  And I uninstalled all those non-functioning screen recorders.

But none of that has the slightest relation to VDH.  VDH has never, does not now, & will never break DRM protection.  As Jérôme said, it would be illegal.
